encryptor/network/client_thread.py

```diff
@@ -100,6 +102,27 @@ def send_message(self, message: str) -> None:
 
         if self._writer is not None:
             # TODO: Don't hardcode encoding here.
-            self._writer.write(Message.of(message.encode("utf-8"), ContentType.BINARY))
+            encrypted_message = encrypt(message.encode("utf-8"), self._mode, self._writer._endpoint_pubkey)
+            self._writer.write(Message.of(encrypted_message, ContentType.BINARY, self._mode))
         else:
             raise RuntimeError("Cannot send a message, writer does not exist")
+
+    @pyqtSlot(str)
+    def send_file(self, filepath: str) -> None:
+        """Send a file to the server."""
+
+        filepath = Path(filepath)
+
+        file = open(filepath, "rb")
+
+        try:
+            if self._writer is not None:
+                data = encrypt(file.read(), self._mode, self._writer._endpoint_pubkey)
+
+                print(f"Sending a file {filepath} to the {self._writer.endpoint_addr}")
+                self._writer.write(Message.of(data, ContentType.FILE, self._mode, filepath.name))
+        finally:
+            file.close()
```

Comment:
`Path` class supports file operations directly, no need to use `open` here.

Reply:
done